README.md

```diff
@@ -183,6 +183,69 @@ By making a contribution to this project, I certify that:
 
 (c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.
 
+## Stability Policy
+
+The most important consideration in every code change is the impact it will have, positive or negative, on the ecosystem (modules and applications). To this end, all Collaborators must work collectively to ensure that changes do not needlessly break backwards compatibility, introduce performance or functional regressions, or negatively impact the usability of the Project on any of the platforms officially targeted for support by the Project. The TSC is responsible for determining the list of supported platforms.
+
+When discussing stability, it is important to first review Node.js' layered architecture:
+
+```
+   +-------------------------------------------+
+   |      Module and Application Ecosystem     |
+   +-------------------------------------------+
+            |                |               |
+   +----------------+        |               |
+   |  Node.js Core  |        |               |
+   |  Library API   |        |               |
+   +----------------+        |               |
+   |     js impl    |        |               |
+   +----------------+        |               |
+            |                |               |
+   +--------------------------------------+  |
+   | Node.js Application Binary Interface |  |
+   +--------------------------------------|  |
+   |              C/C++ impl              |  |
+   +--------------------------------------+  |
+                      |                      |
+        +---------------------------------------+
+        | Dependencies: v8, libuv, openssl, etc |
+        +---------------------------------------+
+```
+
+Node.js currently builds on top of several key dependencies including the V8 Javascript engine, libuv, openssl and others. The Node.js Application Binary Interface provides critical functionality such as the Event Loop which critical to how Node.js operates. The Node.js Core Library is the primary interface through which most Modules and Applications built on top of Node perform I/O operations, manipulate data, access the network, etc. Some modules and applications, however, go beyond the Core Library and bind directly to the Application Binary Interface and dependencies to perform more advanced operations.
+
+Due to the existing layering, modules and applications are sensitive not only to changes in the Core Library API, but the Application Binary Interface and dependendencies as well.
+
+Any API *addition* to either the Node.js Core Library API or Application Binary Interface must result in a *semver-minor* version increase.
```

Comment:
Perhaps something like "Any modification to the Application Binary Interface or Dependencies that would require module or application developers to recompile must result in a *semver-major* version increase."